source/fundamentals/auth.txt

```diff
@@ -52,16 +52,7 @@ Mechanisms
 Default
 ~~~~~~~
```

Comment:
S: Instead of having separate "Default" and "SCRAM-SHA-256" section, I think it makes sense to combine them. I did that in the [kotlin sync PR](https://github.com/mongodb/docs-kotlin-sync/pull/97/files#diff-f68cb6834c0bd8ffa822a698c5e72c79defe0241fe70d10944cc791dd2c6d1a4), which should be basically the same. Otherwise if you do want to keep them separate I'd delete the note about server versions from line 105-108

Reply:
Ahh I forgot to save and commit my auth page changes. But will combine the section, thanks for the suggestion!